### PR TITLE
Amazon Java fixes.

### PR DESCRIPTION
**Pull request by danieljimenez**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -11,7 +11,12 @@
 end
 
 include_recipe 'build-essential'
-include_recipe 'java'
+if platform?('amazon')
+  packages.insert('java-1.8.0-openjdk-devel.x86_64')
```
Comment (mspell-foreflight) [inline recipes/default.rb]: For some reason it looks like my kitchen test is skipping this packages.insert line, so when the execute alternatives line attempts to run, it fails, as the directory doesn't exist. Anybody seeing something similar?

Comment (danieljimenez) [inline recipes/default.rb]: Hm I wasn't sure insert was the right one, but the tests did pass. 🤔

Comment (danieljimenez) [inline recipes/default.rb]: Are you using RVM?

Comment (mspell-foreflight) [inline recipes/default.rb]: Yes, using RVM with Ruby 2.3.1. Interesting that your tests passed. Going to run it again.

Comment (danieljimenez) [inline recipes/default.rb]: Nope, I lied. Mine didn't pass, I'm hooking it to CI now.

**Pull request merged**